packages/storybook/src/button.stories.tsx

```diff
@@ -0,0 +1,39 @@
+import { ArgsTable, Primary, PRIMARY_STORY, Stories } from '@storybook/addon-docs';
+import { Meta, StoryObj } from '@storybook/react';
+import { Button } from '../../components-react/src/Button';
+
+const meta = {
+  title: 'React Component/Button',
+  id: 'react-button',
+  component: Button,
+  args: {
+    children: 'Read more...',
+  },
+  parameters: {
+    tokensPrefix: 'utrecht-button',
+    docs: {
+      page: () => (
+        <>
+          <Primary />
+          <ArgsTable story={PRIMARY_STORY} />
+          <Stories />
+        </>
+      ),
+    },
+  },
+  argTypes: {
+    appearance: {
+      control: { type: 'select' },
+      options: [undefined, 'primary-action-button', 'secondary-action-button', 'subtle-button'],
+    },
```

Comment:
`appearance` is a property here, but there is no implementation of it in the component. Probably less confusing to not show this in Storybook.

Reply:
Removed this.